docs/content-editing/add-a-search-block.md

```diff
@@ -0,0 +1,431 @@
+    ```{image} /content-editing/_static/add-a-search-block-step-30.jpeg
+    :alt: Click "Creation date" and "Effective date" to add them to the list of options to sort by.
+    :target: /_images/add-a-search-block-step-30.jpeg
+    ```
+    +++
+    _Click "Creation date" and "Effective date" to add them to the list of options to sort by._
+    ````
+
+1.  {guilabel}`Save` your page.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-31.jpeg
+    :alt: **[[Save]]** your page.
+    :target: /_images/add-a-search-block-step-31.jpeg
+    ```
+    +++
+    _{guilabel}`Save` your page._
+    ````
+
+1.  The search field will allow the user to search with a text search.
+
+    The facets will narrow that search down.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-32.jpeg
+    :alt: The search field will allow the user to search with a text search.
+    :target: /_images/add-a-search-block-step-32.jpeg
+    ```
+    +++
+    _The search field will allow the user to search with a text search._
+    ````
+
+1.  The Tags facet will allow the user to select items that are tagged and filter according to that tag.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-33.jpeg
+    :alt: The Tags facet will allow the user to select items that are tagged and filter according to that tag.
+    :target: /_images/add-a-search-block-step-33.jpeg
+    ```
+    +++
+    _The Tags facet will allow the user to select items that are tagged and filter according to that tag._
+    ````
+
+1.  The Date range facet will allow the user to narrow down their search inside of a specific date range.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-34.jpeg
+    :alt: The Date range facet will allow the user to narrow down their search inside of a specific date range.
+    :target: /_images/add-a-search-block-step-34.jpeg
+    ```
+    +++
+    _The Date range facet will allow the user to narrow down their search inside of a specific date range._
+    ````
+
+1.  If there are no results in the date range selected, they can clear the range.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-35.jpeg
+    :alt: If there are no results in the date range selected, they can clear the range.
+    :target: /_images/add-a-search-block-step-35.jpeg
+    ```
+    +++
+    _If there are no results in the date range selected, they can clear the range._
+    ````
+
+1.  Clicking on the sorting drop-down will allow the user to sort their search according to a date.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-37.jpeg
+    :alt: Clicking on the sorting drop-down will allow the user to sort their search according to a date.
+    :target: /_images/add-a-search-block-step-37.jpeg
+    ```
+    +++
+    _Clicking on the sorting drop-down will allow the user to sort their search according to a date._
```

Comment:
Avoid the passive voice and impersonal term of "the user". It's better when you speak directly to the reader to do something.

```suggestion
1.  Click on the sorting drop-down to sort the search according to a selected date type.

    ````{card}
    ```{image} /content-editing/_static/add-a-search-block-step-37.jpeg
    :alt: Click on the sorting drop-down to sort the search according to a selected date type.
    :target: /_images/add-a-search-block-step-37.jpeg
    ```
    +++
    _Click on the sorting drop-down to sort the search according to a selected date type._
```